README.rst

```diff
@@ -146,6 +146,8 @@ The following table describes each of the arguments:
  auto_sample                      \+         \+     Override the ``<n_samples>`` value and instead uses chunks of roughly 2.5 MB to improve throughput. Enabled by default.
  encoding                         \+         \+     Specifies dataset encoding. If not provided, the batch_scoring or batch_scoring_sse script attempts to detect the decoding (e.g., "utf-8", "latin-1", or "iso2022_jp"). See the `Python standard encodings <https://docs.python.org/3/library/codecs.html#standard-encodings>`_ for a list of valid values.
  skip_dialect                     \+         \+     Specifies that the script skips CSV dialect detection and uses default "excel" dialect for CSV parsing. By default, the scripts do detect CSV dialect for proper batch generation on the client side.
+ verify_ssl                       \+         \+     Enables SSL verification. If used with a value, e.g. ``--verify_ssl=/path/to/cert``, certificate specified by the path will be used for verification.
```

Comment:
This will confuse users. The default is to enable SSL verification, so saying that its purpose is to enable SSL verification is a little weird. I think we should only support this as an option that takes an argument, the path to the cert list.

And along those lines, I think we should change its name to be something more in line with that usage, i.e. `ssl_verify_cert_path`. We should also specify that it can be a path to a single file or a directory of cert files, per http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification